<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionWidget</class>
 <widget class="QWidget" name="OptionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MenuWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
border: 0;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_OptionVolumeLevel">
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Verdana&quot;;
color: #FFF;
margin-bottom: 6px;</string>
       </property>
       <property name="text">
        <string>Lautstärke einstellen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_OptionVolume">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_OptionVolumeOnOff">
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Verdana&quot;;
color: #FFF;
margin-bottom: 6px;</string>
       </property>
       <property name="text">
        <string>Musik stumm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Verdana&quot;;
color: #FFF;
height: 35px;</string>
       </property>
       <property name="text">
        <string>stumm</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_OptionMusikLoad">
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Verdana&quot;;
color: #FFF;
margin-bottom: 6px;</string>
       </property>
       <property name="text">
        <string>Eigene Musik</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OptionMusikLoad">
       <property name="styleSheet">
        <string notr="true">background-color: #424242;
font: 75 12pt &quot;Verdana&quot;;
color: #FFF;
height: 30px;
width: 150px;
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>Musik laden</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_OptionDesign">
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Verdana&quot;;
color: #FFF;
margin-bottom: 6px;</string>
       </property>
       <property name="text">
        <string>Design</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_OptionDesign">
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #636363;
font: 75 12pt &quot;Verdana&quot;;
color: #FFF;
height: 35px;
padding: 0px;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_OptionSprache">
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Verdana&quot;;
color: #FFF;
margin-bottom: 6px;</string>
       </property>
       <property name="text">
        <string>Sprache</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_OptionSprache">
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #636363;
font: 75 12pt &quot;Verdana&quot;;
color: #FFF;
height: 35px;
padding: 0px;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_OptionVollbild">
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Verdana&quot;;
color: #FFF;
margin-bottom: 6px;</string>
       </property>
       <property name="text">
        <string>Vollbildmodus</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_OptionVollbild">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Verdana&quot;;
color: #FFF;
height: 35px;</string>
       </property>
       <property name="text">
        <string>aktiv</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_OptionBack">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #424242;
font: 75 12pt &quot;Verdana&quot;;
color: #FFF;
height: 50px;
min-width: 250px;
border-radius: 5px;</string>
         </property>
         <property name="text">
          <string>zurück</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
